2996_reflection/reflection.md

```diff
@@ -2235,6 +2275,15 @@ namespace std::meta {
 :::
 :::
 
+### [meta.reflection.info] Reflections
+
+::: bq
+::: addu
+
+[#]{.pnum} The type `std::meta::info` is a synonym for the type produced by the reflection operator ([expr.reflect]{.sref}), and it has the characteristics describes in [basic.types.general]{.sref} and [expr.eq]{.sref}.
```

Comment:
I guess we do say something similar for `std::nullptr_t`... I've never seen that sentence before 😄

Reply:
Yeah, I went looking for `nullptr_t` and was also a bit surprised.  It feels a bit circular...

Reply:
Yeah, I kinda feel like we should not add this sentence - we already define what all the characteristics of `info` are... I also kinda feel like we should remove the `nullptr_t` sentence for being similarly useless... 

It's kinda amusing that _because_ we define it this way, this fails:
```cpp
static_assert(^decltype(nullptr) == ^std::nullptr_t);
```
which is kinda weird.

Reply:
Yes, the alias vs aliased entity distinction is a bit of a pain.

Reply:
Is the code in the synopsis normative enough to allow dropping that sentence?

Reply:
Yeah I think so. I dropped it cause I don't think it adds any information. Also added a missinig "the", otherwise merged.